scripts/photo.js

```diff
@@ -74,15 +74,27 @@ photo.load = function(photoID, albumID) {
 };
 
 photo.update_display_overlay = function () {
+	let showDesc = false; // store the intial value
+
 	lychee.image_overlay = !lychee.image_overlay;
-	if(!lychee.image_overlay)
-	{
+	if(!lychee.image_overlay) {
 		$('#image_overlay').remove();
+	} else {
+		// do this while a photo is showing
+		setInterval(() => {
```

Comment:
**I don't like that.**
I don't want the description and exif to rotate every 3 seconds without my own accord. It induces movement in the corner of the screen, this will attract the attention of the eye and prevent a good reading of the picture.

A better system would be:
- if **Exif** and **Desc** exist:
'click' -> show Exif -> 'click' -> show Desc -> 'click' -> show nothing -> 'click' -> show Exif
- if only **Exif** exist: 
'click' -> show Exif -> 'click' -> show nothing -> 'click' -> show Exif
- if only **Desc** exist: 
'click' -> show Desc -> 'click' -> show nothing -> 'click' -> show Desc

A nice addition would be to bind the key `o` of the keyboard to switch the overlay.

Reply:
Removed.